koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -338,6 +338,15 @@ class BuildContainerTask(BaseTaskHandler):
                     "signing_intent": {
                         "type": ["string", "null"],
                         "description": "Signing intent of the ODCS composes."
+                    },
+                    "skip_build": {
+                        "type": "boolean",
+                        "description": "Skip build, just update buildconfig for autorebuild "
+                                       "and don't start build"
+                    },
+                    "triggered_after_koji_task": {
+                        "type": ["integer", "null"],
```

Comment:
What would `"triggered_after_koji_task": null` mean?

Reply:
basically the same as it wasn't defined, I mean I guess I can have there just 'integer' ?

Reply:
I think I'd prefer just 'integer' for minimum ambiguity.

Reply:
changed